src/clr-angular/utils/drag-and-drop/_drag-and-drop.clarity.scss

```diff
@@ -0,0 +1,22 @@
+@include exports("drag-and-drop.clarity") {
+    .drag-handle {
+        cursor: grab;
+    }
+    .draggable-ghost {
+        position: fixed;
+        visibility: hidden;
+        z-index: 2147483647; // max z-index;
```

Comment:
There is a map called `$clr-layers`. Can we add the `z-index` to that and reference it from there?

Reply:
Please remind me to add something like "All `z-index` properties must use the `$clr-layers` variable because blahblahblah" to our guidelines. You made a very good point raising this.

Or if anyone feels like adding this themselves right now, feel free to do it. 😄

Reply:
This z-index has been moved to `clr-layers`.